### PR TITLE
Task/camera container

### PR DESCRIPTION
**Pull request by TopRamen1**
Added:
- compose file with usage of depthai container for OAK-1 camera
- custom oak1 params
- usage of rviz

### CODE REVIEW

File: demo/compose.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+services:
+  luxonis:
+    image: husarion/depthai:humble-2.7.4
+    volumes:
+      - /dev/bus/usb:/dev/bus/usb
+      - ./config/camera.yaml:/ros2_ws/install/depthai_ros_driver/share/depthai_ros_driver/config/camera.yaml
+    device_cgroup_rules:
+      - 'c 189:* rmw'
+    command: ros2 launch depthai_ros_driver camera.launch.py parent_frame:=oak-1-base-frame
```
Comment (pawelirh) [inline demo/compose.yaml]: I believe we'll stick to the snake_case in the URDF file. If so, we should specify here `oak_1_base_frame`

File: demo/config/camera.yaml
Comment (pawelirh) [inline demo/config/camera.yaml:1]: Is this file specific to the Oak 1 model?
If yes, we should use the camera model in the config file name.

File: demo/compose.yaml (hunk 1 above)
Comment (TopRamen1) [reply demo/compose.yaml]: :+1:

File: demo/config/camera.yaml
Comment (TopRamen1) [reply demo/config/camera.yaml:1]: :+1:

**Review by pawelirh — APPROVED** [review]
LGTM 🆗

**Pull request merged**